types/qaengine/commonqa/commonqa.go

```diff
@@ -152,8 +153,20 @@ func GetPortForService(detectedPorts []int32, qaSubKey string) int32 {
 	return int32(selectedPort)
 }
 
-// IsStateful returns whether the Service should generate a StatefulSet
-func IsStateful(serviceName string) bool {
-	quesKey := common.JoinQASubKeys(common.ConfigServicesKey, serviceName, common.ConfigStatefulSetKey)
-	return qaengine.FetchBoolAnswer(quesKey, fmt.Sprintf("For the service '%s', do you require a StatefulSet instead of a Deployment?", serviceName), nil, false, nil)
+// GetDeploymentType returns whether the Service should generate a StatefulSet
```

Comment:
Can we update this comment too?

Reply:
Resolved in [1608c10](https://github.com/konveyor/move2kube/pull/1098/commits/1608c1008a61017cf59810042d9021b06b034ae7)